packages/@vuepress/theme-default/src/shared/options.ts

```diff
@@ -253,6 +253,11 @@ export interface DefaultThemeLocaleData extends LocaleData {
    */
   openInNewWindow?: string
 
+  /**
+   * Toggle sidebar button a11y text
+   */
+  sidebarButtonText?: string
```

Comment:
Should use `toggleSidebar` to keep align with `toggleDarkMode`, and so does the comment

Reply:
All right